modules/ztp-creating-siteconfig-custom-resources.adoc

```diff
@@ -0,0 +1,351 @@
+    kind: AgentClusterInstall
+    name: <cluster-name>
+    version: v1beta1
+  clusterName: <cluster-name>
+  platform:
+    agentBareMetal:
+      agentSelector:
+        matchLabels:
+          cluster-name: <cluster-name>
+  pullSecretRef:
+    name: assisted-deployment-pull-secret
+----
++
+<1> `base-domain` is the single node cluster’s base domain.
+
+. Create the KlusterletAddonConfig custom resource:
++
+[source,yaml]
+----
+apiVersion: agent.open-cluster-management.io/v1
+kind: KlusterletAddonConfig
+metadata:
+  name: <cluster-name>
+  namespace: <cluster-name>
+spec:
+  clusterName: <cluster-name>
+  clusterNamespace: <cluster-name>
+  clusterLabels:
+    cloud: auto-detect
+    vendor: auto-detect
+  applicationManager:
+    enabled: true
+  certPolicyController:
+    enabled: true <1>
+  iamPolicyController:
+    enabled: true
+  policyController:
+    enabled: true
+  searchCollector:
+    enabled: false
+----
++
+<1> `enabled:` is set to either `true` to enable KlusterletAddonConfig or `false` to disable the KlusterletAddonConfig. Keep `searchCollector` disabled.
+
+. Create the ManagedCluster custom resource:
++
+[source,yaml]
+----
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: <cluster-name>
+spec:
+  hubAcceptsClient: true
+----
+
+. Create the InfraEnv custom resource:
++
+[source,yaml]
+----
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  name: <cluster-name>
+  namespace: <cluster-name>
+spec:
+  clusterRef:
+    name: <cluster-name>
+    namespace: <cluster-name>
+  sshAuthorizedKey: <public-key> <1>
+  agentLabelSelector:
+    matchLabels:
+      cluster-name: <cluster-name>
+  pullSecretRef:
+    name: assisted-deployment-pull-secret
+----
+<1> Enter `public-key` as plain text and use it to SSH into the target bare metal machine when the host is booted from the ISO.
+
+. Create the BareMetalHost custom resource:
++
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: <cluster-name>
+  namespace: <cluster-name>
+  annotations:
+    inspect.metal3.io: disabled
+  labels:
+    infraenvs.agent-install.openshift.io: "<cluster-name>"
+spec:
+  bootMode: "UEFI"
+  bmc:
+    address: <bmc-address> <1>
+    disableCertificateVerification: true
+    credentialsName: <cluster-name>-bmc-secret
+  bootMACAddress: <mac-address> <2>
+  automatedCleaningMode: disabled
+  online: true
+----
+<1> `bmc-address` is the baseboard address of the target bare metal machine.
+<2> `mac-address` is the target bare metal machine’s MAC address.
++
+Optionally, you can add `bmac.agent-install.openshift.io/hostname: <host-name>` as an annotation to set the single node cluster’s host name, otherwise it will default to either a host name from the DHCP server or local host.
+
+. After you have created the custom resources, push the entire directory of generated custom resources to the `Defined Sites` Git Repo.
```

Comment:
Ah, you didn't say to put all of the information into a directory, and I don't see this `Defined Sites` repository (which should be lowercase and not shortened) mentioned anywhere else. Where is this repository?